### PR TITLE
MDS-2415 - Remove 'unknown' fields from NoW application View

### PR DESCRIPTION
**Pull request by mmumby**
- Added 'exp_surface_drill_activity' 
- Commented out all Unknown fields on NoW View
- Updated conditional rendering condition for 'mineralAndCoalAndPlacer' when displaying activities

### CODE REVIEW

File: frontend/src/tests/components/noticeOfWork/__snapshots__/NOWActivities.spec.js.snap [hunk 1]
```diff
@@ -108,25 +108,18 @@ exports[`NOWActivities renders properly 1`] = `
...
+                Describe the proposed reclamation and timing for this specific activity
               </p>
             </Col>
             <Col
               md={12}
               xs={24}
             >
               <p>
-                Unknown
+                N/A
```
Comment (pdittaro) [inline frontend/src/tests/components/noticeOfWork/__snapshots__/NOWActivities.spec.js.snap]: Did we not say that we were going to use dashes?

Comment (mmumby) [reply frontend/src/tests/components/noticeOfWork/__snapshots__/NOWActivities.spec.js.snap]: chatted IRL - let's consult the designers

File: frontend/src/styles/tools/mixins.scss [hunk 2]
```diff
@@ -212,3 +212,7 @@ $x-large-width: 90em;
 .fade-out {
   @include fade-out;
 }
+
```
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/styles/tools/mixins.scss]: Looks like it is not being used, will remove in a follow up PR

File: frontend/src/customPropTypes/noticeOfWork.js [hunk 3]
```diff
@@ -38,41 +38,41 @@ export const nowApplication = shape({
   bcexplosivespermitissued: PropTypes.string,
   bcexplosivespermitnumber: PropTypes.string,
   bcexplosivespermitexpiry: PropTypes.string,
-  campdisturbedarea: PropTypes.string,
+  campdisturbedarea: PropTypes.number,
```
[the remark this answers is not in the document]
Comment (mmumby) [reply frontend/src/customPropTypes/noticeOfWork.js]: On the initial draft, everything was assumed to be a string, after getting a better look at the data I realized that was not the case.

**Pull request merged**